src/plugins/show-tips/tips.ts

```diff
@@ -160,13 +160,50 @@ const tips: TipData[] = [
     learnMore:
       "https://help.desmos.com/hc/en-us/articles/4405901719309-Saving-and-Managing-Graphs",
   },
+  // From flexabrotnt https://discord.com/channels/655972529030037504/663607163532476426/999501013096616036
```

Comment:
```suggestion
  // From flexabrotnt
```

"From flexabrotnt" is enough, discord link isn't needed